src/layouts/components/Homepage/HeroBody.tsx

```diff
@@ -219,9 +227,14 @@ const HeroSideSectionLayout = ({
         <Text textStyle="subhead-1">Alignment</Text>
         <Radio.RadioGroup
           onChange={(nextSectionAlignment) => {
-            setSectionAlignment(nextSectionAlignment as SectionAlignment)
+            onChange({
+              target: {
+                id: "section-0-hero-alignment",
+                value: nextSectionAlignment,
+              },
+            })
           }}
-          defaultValue="left"
+          defaultValue={alignment || "left"}
```

Comment:
nit: given that alignment already has a default, we don't need to use `left` as a fallback

Reply:
removed